seqr/views/utils/file_utils_tests.py

```diff
@@ -96,15 +106,50 @@ def test_temp_file_upload(self):
             'parsedData': PARSED_DATA,
             'uploadedFileId': mock.ANY,
         })
+        # Remove temp file
+        load_uploaded_file(response_json['uploadedFileId'])
```

Comment:
why is this necessary? everything gets written to temp directories so its not a big deal if the tests leave some files there

Reply:
I'm not sure about it. Delete these lines.